Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/compositeeditor/CompositeEditorModel.java

```diff
@@ -867,9 +867,8 @@ public boolean isEditComponentAllowed() {
 			baseDt = comp.getDataType();
 			baseDt = DataTypeUtils.getBaseDataType(baseDt);
 		}
-		return ((baseDt != null) && !(baseDt instanceof BuiltInDataType) &&
-			!(baseDt instanceof MissingBuiltInDataType) &&
-			((baseDt instanceof Structure) || baseDt instanceof Union || baseDt instanceof Enum));
+		return !(baseDt instanceof BuiltInDataType) && (baseDt instanceof Structure
+				|| baseDt instanceof Union || baseDt instanceof Enum);
```

Comment:
This should be covered by `!(baseDt instanceof BuiltInDataType)` because MissingBuiltInDataType impelements the interface BuiltInDataType as you can in the UML diagram: 
![image](https://user-images.githubusercontent.com/29959150/79094629-70b92f80-7d58-11ea-9f81-ab4b845397d7.png)